### PR TITLE
Notification sound fix

### PR DESCRIPTION
**Pull request by vanowm**
Fix for #277

### CODE REVIEW

**Comment by OrangeDrangon** [comment]
This change makes no sense. We are overriding the Notification constructor which is expected to return a notification object and you are instead returning a boolean when we should be returning the Notification in case the google code wants to do more with it.

**Comment by vanowm** [comment]
The second (non-system) sound is done by google's routine which calls the notification. By returning boolean we are telling google not to play sound.

**Comment by OrangeDrangon** [comment]
By returning a boolean from a constructor for a notification you are violating the API. I find it hard to believe this works with no unintended consequences though I have not tested it.

**Comment by vanowm** [comment]
I see. So by returning boolean I'm actually forcing google's code to fail. Specifically it fails while attempting set click event.
Not sure if it's even possible stop it from making sound other way.

File: src/bridge.ts [hunk 1]
```diff
@@ -105,25 +105,30 @@ window.Notification = function (title: string, options: NotificationOptions) {
...
+  const isSound = settings.get(SETTING_NOTIFICATION_SOUND,true) as boolean;
   notificationOpts.silent = true;
 
   const notification = new ElectronNotification(notificationOpts);
   notification.addListener("click", () => {
     app.mainWindow?.show();
     document.dispatchEvent(new Event("focus"));
+    if (!isSound)
+    {
+      (document.querySelector('a[href$="/' + options.data.id + '"]') as HTMLElement).click();
```
Comment (OrangeDrangon) [inline src/bridge.ts:117]: an explanation of what this line is doing would be nice in a short comment or two

**Pull request closed without merging**